### ISSUE TITLE
How to deploy during development

### ISSUE THREAD

**Issue by gcobr** [Sundog-Interactive/DF14-Visualforce-AngularJS-RemoteObjects#1]
All this looks pretty cool, but how do you work during development?

Do you have to zip everything and upload the file as a static resource through the web UI every time you make a little change? If that's the process, it seems very unproductive.

**Comment by kulishch** [comment]
@gcobr yes, for this project I would zip up the project and upload it to a static resource in Salesforce. Definitely not the most productive workflow.

My actual development workflow uses Joe Ferraro's MavensMate (https://github.com/joeferraro/MavensMate) to pull down the org locally and then create a resource bundle (http://www.joe-ferraro.com/2012/12/mavensmate-resource-bundles/) of the static resource where I can manipulate the files individually and then deploy the bundle as a static resource back to the org. Works very nice.

I've also considered writing a Grunt plugin using JSForce (https://jsforce.github.io/) that would allow you to zip a project and push it up as a static resource to an org without having to pull down the whole org. This would only be useful if you were to write an SPA that was agnostic of the fact it was running in a Visualforce page as well as making sure dependencies are properly injected and mocked for testing.

**Comment by SindhuMani** [comment]
@kulishch Great Work! Well Appreciated. How can i use the code in my developer instance and execute ?
Because i would like to demo the same to my team to show how angularJS works. Please help

**Comment by kulishch** [comment]
@SindhuMani you'll need to generate the ZIP file using the Grunt script and upload it to a Static Resource called ContactManagerSPA. Then you'll need to create a Visualforce page and paste in the markup from the contact_manager.page. Then access the browse to the Visualforce page to demo the code.
